pkg/cloud/azure/actuators/machine/actuator_test.go

```diff
@@ -343,7 +343,7 @@ func TestReconcileVMSuceededState(t *testing.T) {
 		t.Errorf("failed to create machine: %+v", err)
 	}
 
-	if fakeVMService.GetCallCount != 1 {
+	if fakeVMService.GetCallCount != 2 {
 		t.Errorf("expected get to be called just once")
```

Comment:
called twice? 

To check I'm following: this is to reflect the create + update call, right?